core/src/test/scala/unit/kafka/server/ShareGroupHeartbeatRequestTest.scala

```diff
@@ -894,7 +894,7 @@ class ShareGroupHeartbeatRequestTest(cluster: ClusterInstance) {
       val allPartitionsMetadata = waitForAllPartitionsMetadata(brokersToValidate, topic, totalPartitionCount)
       (0 until totalPartitionCount - 1).foreach(i => {
         allPartitionsMetadata.get(new TopicPartition(topic, i)).foreach { partitionMetadata =>
-          assertEquals(totalPartitionCount, partitionMetadata.replicas.size)
+          assertEquals(totalPartitionCount, partitionMetadata.isr.size)
```

Comment:
This is not exactly the same behavior, but the updated behavior is fine for the test (since isr and replicas is the same). The reason for the change is that the replicas are no longer returned by the method being used here.